test/unit/BondingManager.js

```diff
@@ -24,6 +25,8 @@ contract("BondingManager", accounts => {
         fixture = new Fixture(web3)
         await fixture.deploy()
 
+        const ll = await LinkedList.new()
+        BondingManager.link("SortedDoublyLL", ll)
```

Comment:
I think this needs to be:

```
BondingManager.link("SortedDoublyLL", ll.address)
```

Reply:
ah yes you're right , shouldn't have done this in between other stuff. fixed now.